src/timer.rs

```diff
@@ -294,6 +294,12 @@ pub struct TimerError {
     desc: &'static str,
 }
 
+impl fmt::Display for TimerError {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        write!(fmt, "{}", self.desc)
```

Comment:
It would be nice to include the `TimerErrorKind` in this output as well, not just the description.